package.json

```diff
@@ -24,5 +24,6 @@
   },
   "files": [
     "id.js"
```

Comment:
If we're publishing to npm, what do we think about also including the README.md (the specification) along with it?